eng/README.md

```diff
@@ -49,9 +49,21 @@ release branch is created, someone with `darc` installed locally will need to
 run the `add-subscription` command to configure updates against that new branch.
 
 
+#### Build Asset Manifest Promotion
+
+Builds from main and release branches will push NuGet package metadata to the
+darc/maestro Build Asset Registry.  This build information will also be promoted
+to a default darc/maestro channel if one is configured.  Default channels are
+manually managed at this time.  To configure a new default repo+branch <-> channel
+association, run the [`darc add-default-channel`][6] command:
+```
+darc add-default-channel --channel ".NET 6" --branch "main" --repo https://github.com/xamarin/xamarin-macios
+```
```

Comment:
Should you add instructions for release branches, too?

Reply:
Updated